request/request.go

```diff
@@ -69,12 +69,27 @@ func (c *RequestClient) Post(ctx context.Context,
 		return nil, err
 	}
 
-	return c.makeRequestAndParseResult(req, result)
+	return c.makeJSONRequestAndParseResult(req, result)
+}
+
+// Delete makes a Delete request to the desired URL for plain text requests.
+func (c *RequestClient) Delete(ctx context.Context,
+	url string, headers ...Header,
+) (*http.Response, error) {
+	allHeaders := c.DefaultHeaders
+	allHeaders = append(allHeaders, headers...)
+
+	req, err := makeDeleteRequest(ctx, url, allHeaders)
+	if err != nil {
+		return nil, fmt.Errorf("error creating request: %w", err)
+	}
+
+	return c.makePlainTextRequestAndParseResult(req)
 }
 
 var ErrNone200Status = errors.New("error response from API")
 
-func (c *RequestClient) makeRequestAndParseResult(req *http.Request, result any) (*http.Response, error) {
+func (c *RequestClient) makeJSONRequestAndParseResult(req *http.Request, result any) (*http.Response, error) {
```

Comment:
This is more like `makeRequestAndParseJSONResult`, since it's parsing a JSON result.